apps/ecommerce/frontend/src/providers/ResourceFieldProvider.tsx

```diff
@@ -4,9 +4,9 @@ import { ReactNode } from 'react';
 export type ResourceFieldProviderProps = {
   isMultiple: boolean;
   handleAddResource: () => Promise<any[]>;
-  handleRemove: (index: number) => void;
-  handleMoveToTop?: (index: number) => void;
-  handleMoveToBottom?: (index: number) => void;
+  handleRemove: (index?: number) => void;
```

Comment:
Repeat comment for completeness sake: Is the index now optional so that it works for both multiple and single?